src/components/NavBar/nav-bar.scss

```diff
@@ -0,0 +1,4 @@
+.nav-bar{
+    z-index: 9999;
```

Comment:
no, it shouldn't be fixed (also, if you wanted it to be fixed you could simply pass `fixed={true}` to it ;) )
what I mean is: as it it doesn't have `position: absolute` nor `position: fixed` the property `z-index` is useless -> please remove it :)